osgi-dependencies/aet-features.xml

```diff
@@ -126,4 +129,41 @@
         <bundle>mvn:commons-beanutils/commons-beanutils/1.9.2</bundle>
         <!--GRID FS bundles-->
     </feature>
+
+    <feature name="aet-w3c-validator" version="0.6.0" description="Dependencies for AET W3C validator" install="auto">
+        <!-- these were used from cxf in old Karaf setup -->
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xmlbeans/2.6.0_2</bundle>
+    </feature>
+
+    <feature name="aet-bundles" version="0.6.0" description="AET Bundles" install="auto">
+
+        <feature dependency="true" prerequisite="true">aet-scr-and-war</feature>
+        <feature dependency="true" prerequisite="true">aet-cxf</feature>
+        <feature dependency="true" prerequisite="true">aet-activemq</feature>
+        <feature dependency="true" prerequisite="true">aet-crypto</feature>
+        <feature dependency="true" prerequisite="true">aet-apache-commons</feature>
+        <feature dependency="true" prerequisite="true">aet-guice</feature>
+        <feature dependency="true" prerequisite="true">aet-jackson</feature>
+        <feature dependency="true" prerequisite="true">aet-dependencies</feature>
+
+        <bundle>file:./../aet_bundles/current/com.cognifide.aet.communication-api-2.1.4-SNAPSHOT.jar</bundle>
```

Comment:
Could we somehow have the `{aet-version}` variable used here?

Reply:
This will be done with another pull request.